sdcm/cluster.py

```diff
@@ -35,7 +35,7 @@
 from avocado.utils import script
 from datetime import datetime
 from .log import SDCMAdapter
-from .remote import Remote
+from .remote import Remote, RemoteFabric
```

Comment:
if Remote isn't used anymore ? maybe we should remove it ?

Reply:
we still want to support the "old" way during the transition period

Reply:
o.k. didn't yet seen a conditional usage below. but fair enough